.dockerignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
Let's switch this to excluded format instead, so we ignore everything `*` and add just the files we want to include (with `!`).